var/spack/repos/builtin/packages/graphviz/package.py

```diff
@@ -74,6 +76,7 @@ class Graphviz(AutotoolsPackage):
     patch('https://raw.githubusercontent.com/easybuilders/easybuild-easyconfigs/master/easybuild/easyconfigs/g/Graphviz/Graphviz-2.40.1_icc_vmalloc.patch',
           sha256='813e6529e79161a18b0f24a969b7de22f8417b2e942239e658b5402884541bc2',
           when='@:2.40%intel')
+    patch('ps2pdf.patch')
```

Comment:
Since this is always patching the `configure.ac` file, then `autoreconf` should always be invoked, right? There are some perhaps-stale comments and logic for that.

Reply:
I think we always run autoreconf for this package since we always check out from git commits.